### PR TITLE
Implement delete to boulder controller

### PR DESCRIPTION
**Pull request by Crazmiss**

### CODE REVIEW

File: app/Http/Controllers/BoulderController.php [hunk 1]
```diff
@@ -99,11 +99,13 @@ public function update(Request $request, $id)
...
      *
      * @param int $id
-     * @return \Illuminate\Http\Response
+     * @return \Illuminate\Http\JsonResponse
+     * @throws \Exception
      */
     public function destroy($id)
     {
-        //
+        $success = Auth::user()->boulders()->find($id)->delete();
```
Comment (Boydbueno) [inline app/Http/Controllers/BoulderController.php]: A bit more efficient in this case would be to do:
`Auth::user()->boulders()->destroy($id)`
Then you don't first retrieve the boulder `find($id)` before you delete it.

File: app/Http/Controllers/BoulderController.php [hunk 2]
```diff
@@ -99,11 +99,13 @@ public function update(Request $request, $id)
...
+        return response()->json($success);
```
Comment (Boydbueno) [inline app/Http/Controllers/BoulderController.php]: It's quite common to not return anything after a delete. So basically just a 204 response without body.
You can achieve that with `return response()->noContent();`

File: app/Http/Controllers/BoulderController.php (hunk 1 above)
Comment (Crazmiss) [reply app/Http/Controllers/BoulderController.php]: ah ok that makes now also more sense with the name destroy of the call

File: app/Http/Controllers/BoulderController.php (hunk 2 above)
Comment (Crazmiss) [reply app/Http/Controllers/BoulderController.php]: ah that also is cool!

File: app/Http/Controllers/BoulderController.php (hunk 1 above)
Comment (Crazmiss) [reply app/Http/Controllers/BoulderController.php]: I dont think this works 
i get an error on this..

Comment (Boydbueno) [reply app/Http/Controllers/BoulderController.php]: That's odd. Not important enough to find out why/how. :P

**Pull request merged**